docs/index.jsx

```diff
@@ -20,11 +20,11 @@ const images = [{
 
 ReactDom.render(
     <SuiCard className={'myCustom-class'}
-      primary={
-        <a href={images[0].link}><img src={images[0].src} /></a>
-      }
+      primary={[
+        <a key={1} href={images[0].link}><img src={images[0].src} /></a>
```

Comment:
I think that the sui-card should be the one that gives keys to its components, don't you?